### PR TITLE
combining Fest and About page

### PR DESCRIPTION
**Pull request by Subaharish**
Signed-off-by: Subaharish <48942935+Subaharish@users.noreply.github.com>

### CODE REVIEW

File: functions/views/about.ejs [hunk 1]
```diff
@@ -114,59 +107,52 @@
...
+											A logo makes a significant impact depending on public perception
+											Our logo has a beautiful peacock with colorful
+											feathers.<br> Peacock is a symbol of beauty and grace. The colorful feathers
+											of
+											the peacock depict how people with their different and colorful perspectives
+											and personalities
+											can come together and form a more colorful and better world. It depicts
+											togetherness even in differences.
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_2">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:143]: Label tag is there

File: functions/views/about.ejs [hunk 2]
```diff
@@ -175,66 +161,212 @@
...
+											MUTLI FEST - AYKYA”.
+											<br />Our motive is to handle
+											such unprecedented changes
+											through a delightful platform.
+											AYKYA is a synergy of all the
+											protons of Amrita Bengaluru
+											campus (B.Tech, M.Tech and MBA
+											students).
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:233]: Label tag is there

File: functions/views/about.ejs [hunk 3]
```diff
@@ -175,66 +161,212 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_4">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:342]: Label tag is there

File: functions/views/about.ejs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:143]: Label tag is there

File: functions/views/about.ejs [hunk 4]
```diff
@@ -175,66 +161,223 @@
...
+											MUTLI FEST - AYKYA”.
+											<br />Our motive is to handle
+											such unprecedented changes
+											through a delightful platform.
+											AYKYA is a synergy of all the
+											protons of Amrita Bengaluru
+											campus (B.Tech, M.Tech and MBA
+											students).
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:233]: Label tag is there

File: functions/views/about.ejs [hunk 5]
```diff
@@ -175,66 +161,223 @@
...
+											extravaganza that helped unleash hidden talent, confront the weariest of
+											fears and enhance the creativity of minds all throughout this tough time of
+											lockdown, in the midst of a heart wrenching pandemic.<br>
+
+											Our theme for this year was mental health.
+											A subtle and an unvoiced subject, began to catch everyone’s
+											eye, gaining the right to spread awareness and prominence, in order to
+											normalise the matter as any other.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_4">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:289]: only one label tag is there

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:289]: Label tag is there

File: functions/views/about.ejs [hunk 6]
```diff
@@ -175,66 +161,223 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_4">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:342]: only one label tag is there

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:342]: Label tag is there

File: functions/views/about.ejs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:143]: Label tag is there

File: functions/views/about.ejs (hunk 4 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:233]: Label tag is there

File: functions/views/about.ejs (hunk 5 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:289]: Label tag is there

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:289]: Label tag is there

File: functions/views/about.ejs (hunk 6 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:342]: only one label tag is there

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:342]: Label tag is there

File: functions/views/about.ejs [hunk 7]
```diff
@@ -60,113 +53,99 @@
...
+											A logo makes a significant impact depending on public perception
+											Our logo has a beautiful peacock with colorful
+											feathers.<br> Peacock is a symbol of beauty and grace. The colorful feathers
+											of
+											the peacock depict how people with their different and colorful perspectives
+											and personalities
+											can come together and form a more colorful and better world. It depicts
+											togetherness even in differences.
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:133]: Label tag is present

File: functions/views/about.ejs [hunk 8]
```diff
@@ -175,319 +154,212 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:305]: Label tag is present

File: functions/views/about.ejs [hunk 9]
```diff
@@ -175,319 +154,201 @@
...
+											MUTLI FEST - AYKYA”.
+											<br />Our motive is to handle
+											such unprecedented changes
+											through a delightful platform.
+											AYKYA is a synergy of all the
+											protons of Amrita Bengaluru
+											campus (B.Tech, M.Tech and MBA
+											students).
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_2">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:209]: Label tag is present

File: functions/views/about.ejs [hunk 10]
```diff
@@ -175,319 +154,201 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:305]: Label tag is present

File: functions/views/about.ejs (hunk 9 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:209]: Label tag is present

File: functions/views/about.ejs (hunk 10 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:305]: Label tag is present

File: functions/views/about.ejs (hunk 9 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:209]: Label tag is present

File: functions/views/about.ejs (hunk 10 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:305]: Label tag is present

File: functions/views/about.ejs [hunk 11]
```diff
@@ -60,113 +53,96 @@
...
+											A logo makes a significant impact depending on public perception
+											Our logo has a beautiful peacock with colorful
+											feathers.<br> Peacock is a symbol of beauty and grace. The colorful feathers
+											of
+											the peacock depict how people with their different and colorful perspectives
+											and personalities
+											can come together and form a more colorful and better world. It depicts
+											togetherness even in differences.
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:133]: Label tag is present

File: functions/views/about.ejs [hunk 12]
```diff
@@ -175,319 +151,197 @@
...
+											MUTLI FEST - AYKYA”.
+											<br />Our motive is to handle
+											such unprecedented changes
+											through a delightful platform.
+											AYKYA is a synergy of all the
+											protons of Amrita Bengaluru
+											campus (B.Tech, M.Tech and MBA
+											students).
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_2">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:209]: Label tag is present

File: functions/views/about.ejs [hunk 13]
```diff
@@ -175,319 +151,197 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:305]: Label tag is present

**Review by VinayakTekade — CHANGES_REQUESTED** [review]
@Subaharish Change the layout of images and along with the text so that images are clearer.

**Comment by Subaharish** [comment]
> @Subaharish Change the layout of images and along with the text so that images are clearer. Don't worry about CodeFactor issues. I will fix it.

@VinayakTekade Resized the images and Changed the font color

File: functions/views/about.ejs [hunk 14]
```diff
@@ -175,319 +154,212 @@
...
+											MUTLI FEST - AYKYA”.
+											<br />Our motive is to handle
+											such unprecedented changes
+											through a delightful platform.
+											AYKYA is a synergy of all the
+											protons of Amrita Bengaluru
+											campus (B.Tech, M.Tech and MBA
+											students).
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_2">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:198]: label tag is present

File: functions/views/about.ejs [hunk 15]
```diff
@@ -175,319 +140,197 @@
...
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:198]: label tag is present

File: functions/views/about.ejs [hunk 16]
```diff
@@ -175,319 +140,197 @@
...
+											Rise from the ashes, A phrase derived from the mythological phoenix, is
+											often used to describe the innate behaviour of humans. A reminder that we
+											too can be reborn from the shells of our former selves. We believed that
+											this tagline was apt for starting our conversation into mental health -
+											holding high the ‘never give up’ outlook on obstacles, resilience to
+											hardships, grit to match the draining circumstances and over all, the power
+											to rebound with twice the strength and emerge victorious no matter how
+											trying the situation might be.<br>
+										</p>
+										<input type="checkbox" name="button" id="collapse-btn_3">
```
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:294]: label tag is present

File: functions/views/about.ejs (hunk 15 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:198]: Label tag is present

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:198]: Label tag is present

File: functions/views/about.ejs (hunk 16 above)
[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:294]: Label tag is present

[the remark this answers is not in the document]
Comment (Subaharish) [reply functions/views/about.ejs:294]: Label tag is present

**Pull request merged**